maven-help-plugin/src/main/java/org/apache/maven/plugins/help/EvaluateMojo.java

```diff
@@ -211,7 +211,7 @@ public void execute()
         {
             while ( true )
             {
-                getLog().info( "Enter the Maven expression i.e. ${project.groupId} or 0 to exit?:" );
+                System.out.println( "Enter the Maven expression i.e. ${project.groupId} or 0 to exit?:" );
```

Comment:
That is supposed to be `print` and not `println`. Otherwise you have an ugly line break.